com.liferay.blade.cli/src/com/liferay/blade/cli/DeployCommand.java

```diff
@@ -169,80 +170,91 @@ private void installOrUpdate(File outputFile) throws Exception {
 			isFragment = fragmentHost != null;
 
 			bsn = bundle.getBsn();
+
+			if(isFragment) {
+				hostBSN =
+						new Parameters(fragmentHost).keySet().iterator().next();
```

Comment:
in our ide side code , we just use 
`ragmentHostName.trim().substring( 0, fragmentHostName.indexOf( ";bundle-version" ) );`
to get the hostBSN, so actually it is the same principle.
but I can't think of any parameter that could be the first one before hostBSN,